src/components/BookForm.js

```diff
@@ -0,0 +1,30 @@
+import React from 'react';
+
+const categories = ['Action', 'Biography', 'History', 'Horror', 'Kids', 'Learning', 'Sci-Fi'];
+
+const getCategories = () => {
+  const result = [];
+  for (let i = 0; i < categories.length; i += 1) {
+    result.push(
+      <option key={categories[i]} value={categories[i]}>
+        {categories[i]}
+      </option>,
+    );
+  }
+  return result;
+};
+
+const BookForm = () => (
+  <form>
+    <label htmlFor="title">
```

Comment:
- Please remember that the BookForm component is going to be connected with Redux and, therefore, should be placed at `src/containers` directory